packages/zotero.rb

```diff
@@ -0,0 +1,127 @@
+# Thanks to @agkphysics of AUR repository for his zotero pkgbuild, which heavily based this package.
+
+require 'package'
+
+class Zotero < Package                 # The first character of the class name must be upper case
+  description 'A free, easy-to-use tool to help you collect, organize, cite, and share your research sources..'
+  homepage 'https://www.zotero.org/'
+  version '6.0.26'
+  license 'AGPL3' # license of source
+  compatibility 'x86_64'
+  source_url 'https://aur.archlinux.org/cgit/aur.git/plain/zotero.desktop?h=zotero'
+  source_sha256 'eab76db7a56a4d9aaa17baaf240b82fcf57944a4ddf8ef1b58cc64182426cedc'   # Use the command "sha256sum"
+
+  depends_on 'dbus_glib'            # packages required by this package
+  depends_on 'gtk3'
+  depends_on 'desktop_file_utils'
+  depends_on 'nss'
+  depends_on 'libxt'
+  depends_on 'perl'
+  depends_on 'node' => :build      # packages with "=> :build" are only required if you're building from source 
+  depends_on 'git' => :build
+  depends_on 'zip' => :build
+  depends_on 'unzip' => :build
+  depends_on 'python3' => :build
+  depends_on 'curl' => :build
+  depends_on 'wget' => :build
+  depends_on 'rsync' => :build
+  depends_on 'nodebrew' => :build
+
+  def self.build                   # the steps required to build the package
+    $srcdir = `realpath . | tr -d '\n'`
+
+    system "git clone -b 6.0.26 https://github.com/zotero/zotero.git"
```

Comment:
You want something like this for every time you do a git clone, with a variable for the branch or tag, so it is easy to update:
```
system "git clone -b #{version} https://github.com/zotero/zotero.git"
```